cmd/sfu/main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"syscall"
 
 	"github.com/matrix-org/waterfall/pkg/config"
+	"github.com/matrix-org/waterfall/pkg/profiling"
```

Comment:
What's the benefit to moving the main file?

Reply:
- In pkg are public library file. main.go is not a library file.
- You setup in main.go the SFU. You can have several mains with different setups
For example a SFU uses RPC as the signaling layer, another SFU use Data-Channels.